src/stateless/demo_data.js

```diff
@@ -0,0 +1,25 @@
+
+export const defaultItems = [
```

Comment:
We should put some thought into generating edge cases here. I'm not saying the data is missing some cases, I'm just thinking if we covered all edge cases...